src/HotChocolate/Language/src/Language.Rewriters/HotChocolate.Language.Rewriters.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>HotChocolate.Language.Rewriters</PackageId>
+    <AssemblyName>HotChocolate.Language.Rewriters</AssemblyName>
+    <RootNamespace>HotChocolate.Language.Rewriters</RootNamespace>
+    <Description>This package contains Syntax Rewrite primitives of Hot Chocolate.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="HotChocolate.Language.Rewriters.csproj.DotSettings" />
+  </ItemGroup>
+
+  <ItemGroup>
+   <ProjectReference Include="..\..\..\Core\src\Abstractions\HotChocolate.Abstractions.csproj" />
```

Comment:
Language is not allowed to reference anything else ... the exception is Utilities.